src/main/resources/application.yaml

```diff
@@ -36,13 +36,13 @@ spring:
             client-secret: internal
 idam:
   api:
-    url: ${IDAM_URL:https://idam-web-public.aat.platform.hmcts.net}
+    url: ${IDAM_URL:http://localhost:5000}
   s2s-auth:
     microservice: aac_mca
     totp_secret: ${MANAGE_CASE_S2S_KEY:AAAAAAAAAAAAAAAA}
     url: ${S2S_URL:http://localhost:4502}
   s2s-authorised:
-    services: ${MANAGE_CASE_S2S_AUTHORISED_SERVICES:ccd_data}
+    services: ${MANAGE_CASE_S2S_AUTHORISED_SERVICES:ccd_gw}
```

Comment:
just noticed, sorry: xui_webapp - just for consistency and to avoid confusion later